core/commands/pin.go

```diff
@@ -15,6 +16,14 @@ import (
 	context "gx/ipfs/QmZy2y8t9zQH2a1b8q2ZSLKp17ATuJoCNxxyMFG5qFExpt/go-net/context"
 )
 
+var (
+	tDirect    = "direct"
+	tIndirect  = "indirect"
+	tRecursive = "recursive"
+	tAll 			 = "all"
```

Comment:
run go fmt on all of these, the formatting is off at least here (maybe elsewhere too)